test/concurrent/emit-interrupt-fail.cc

```diff
@@ -23,29 +23,27 @@ TYPED_TEST(ConcurrentTypedTest, EmitInterruptFail) {
                .raises<std::runtime_error>()
                .interruptible()
                .begin([](auto& k, Interrupt::Handler& handler) {
-                 handler.Install([&k]() {
-                   k.Fail(std::runtime_error("error"));
-                 });
+                 handler.Install(
+                     [&k]() { k.Fail(std::runtime_error("error")); });
                  k.Begin();
                })
-               .next([i = 0](auto& k) mutable {
+               .next([i = 0](auto& k, Interrupt::Handler& handler) mutable {
                  i++;
                  if (i == 1) {
                    k.Emit(i);
                  }
                })
-        | this->ConcurrentOrConcurrentOrdered([]() {
-            return Map([](int i) {
-              return std::to_string(i);
-            });
-          })
+
+        | this->ConcurrentOrConcurrentOrdered(
+            []() { return Map([](int i) { return std::to_string(i); }); })
```

Comment:
Please revert this formatting change. And please review your PRs in the future to undo mistakes like this so I don't have to! Thank you!